### DISCUSSION TITLE
Add option to disable fields instead of hiding them

### DISCUSSION THREAD

**Discussion by mikesnoeren** [craftcms/cms#16201] [Ideas] [answered]
We often encounter scenarios where users need to choose between two inputs, for example; an Video file (asset) or a YouTube video (URL). This is how the conditions for that would look:

- Show Field X if Field Y has a value.
- Show Field Y if Field X has a value.

This setup causes the UI to become quite jittery as fields appear and disappear dynamically. It would be much more user-friendly to have an option to disable fields instead of completely hiding them, ensuring smoother interactions and a clearer interface.

**Comment by piotrpog** [comment]
Agreed. This is something i have suggested here:
https://github.com/craftcms/cms/discussions/15903

**Comment by brandonkelly** [comment, marked as answer]
Closing in favor of #15903.
